.github/workflows/nightly-builds.yml

```diff
@@ -0,0 +1,34 @@
+name: Nightly Build
+
+on:
+  schedule:
+    - cron: "21 0 * * *"
+  workflow_dispatch:
+
+permissions: {}
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Compile, Code Style, Binary Compatibility
+    runs-on: ubuntu-20.04
+    env:
+      JAVA_OPTS: -Xms2G -Xmx2G -Xss2M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: coursier/setup-action@v1.2.1
+        with:
+          jvm: adopt:8
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
+      - name: "Run all tests. Run locally with: sbt +~2.13 \"test\""
+        run: sbt +~2.13 "test"
```

Comment:
What does this `~` syntax do exactly?

Reply:
not sure - I just copied it from the existing workflow yml - I can remove it

Reply:
I would say leave it in there, was just curious. Ill approve the PR